runtime/compiler/runtime/MetaData.cpp

```diff
@@ -226,6 +229,52 @@ createExceptionTable(
       }
    }
 
+#if defined(J9VM_OPT_MICROJIT)
+static void
+createMJITExceptionTable(
+      TR_MethodMetaData * data,
+      MJIT_ExceptionTableEntryIterator & exceptionIterator,
+      bool fourByteOffsets,
+      TR::Compilation *comp)
+   {
+   uint8_t * cursor = (uint8_t *)data + sizeof(TR_MethodMetaData);
+
+   for (TR_ExceptionTableEntry * e = exceptionIterator.getFirst(); e; e = exceptionIterator.getNext())
+      {
+      if (fourByteOffsets)
+         {
+         *(uint32_t *)cursor = e->_instructionStartPC, cursor += 4;
+         *(uint32_t *)cursor = e->_instructionEndPC, cursor += 4;
+         *(uint32_t *)cursor = e->_instructionHandlerPC, cursor += 4;
+         *(uint32_t *)cursor = e->_catchType, cursor += 4;
```

Comment:
There's no need to use the comma operator here, and the statements should be on separate lines:
```
  *(uint32_t *)cursor = e->_instructionStartPC;
  cursor += 4;
```
Likewise, several places below.

Reply:
Thanks again for the review Keith. This code was created because we needed a function to work on the `MJIT_ExceptionTableEntryIterator`. There is a nearly identical one used by TR above which was used as a starting point for this. The above lines of code were copied from that location.

Should I leave this code a is, modify both, or only make the change in the `createMJITExceptionTable` version?

Reply:
I'll let someone more familiar with the existing code comment on whether it should be updated, but my suggestion is to avoid the comma operator in new code.